.expeditor/config.yml

```diff
@@ -5,5 +5,66 @@ slack:
   notify_channel: supermarket
 
 github:
-  enforce_workflow: true
+  # The file where the MAJOR.MINOR.PATCH version is kept. The version in this file
+  # is bumped automatically via the `built_in:bump_version` merge_action.
+  version_file: "VERSION"
+  # The file where our CHANGELOG is kept. This file is updated automatically with
+  # details from the Pull Request via the `built_in:update_changelog` merge_action.
+  changelog_file: "CHANGELOG.md"
+  # The Github Team primarily responsible for handling incoming Pull Requests.
   maintainer_group: chef/supermarket
+
+# These actions are taken, in order they are specified, anytime a Pull Request is merged.
+merge_actions:
+  - built_in:bump_version:
+      ignore_labels:
+        - "Version: Skip Bump"
+        - "Expeditor: Skip All"
+  - built_in:update_changelog:
+      ignore_labels:
+        - "Changelog: Skip Update"
+        - "Expeditor: Skip All"
+  - built_in:trigger_omnibus_build:
+      ignore_labels:
+        - "Omnibus: Skip Build"
+        - "Expeditor: Skip All"
+      only_if: built_in:bump_version
```

Comment:
Interpretation verification: if a PR is labeled `Version: Skip Bump`, in addition to not incrementing the version number, it will also _not_ trigger a build. Right? :D